src/libs/actions/IOU.ts

```diff
@@ -1150,6 +1152,23 @@ function buildOnyxDataForTrackExpense(
                 },
             },
         );
+
+        if (actionableTrackExpenseWhisper && !iouReport) {
+            optimisticData.push({
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${chatReport?.reportID}`,
+                value: {
+                    [actionableTrackExpenseWhisper.reportActionID]: actionableTrackExpenseWhisper as OnyxTypes.ReportAction,
+                },
+            });
+            optimisticData.push({
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.REPORT}${chatReport.reportID}`,
+                value: {
+                    lastVisibleActionCreated: actionableTrackExpenseWhisper.created,
```

Comment:
I think we also need to set `lastMessageText` (and `lastMessageHtml`). Otherwise, LHN subtitle will not show the correct status.
![Screenshot 2024-05-13 at 10 30 04 PM](https://github.com/Expensify/App/assets/3004659/e372112e-22db-43ca-aeb5-f4fc23cd7b61)